db.go

```diff
@@ -402,21 +401,29 @@ func (db *DB) hasSyncedFreelist() bool {
 	return db.meta().Freelist() != common.PgidNoFreelist
 }
 
+func (db *DB) fileSize() (int, error) {
+	info, err := db.file.Stat()
+	if err != nil {
+		return 0, fmt.Errorf("file stat error: %w", err)
+	}
+	sz := int(info.Size())
+	if sz < db.pageSize*2 {
+		return 0, fmt.Errorf("file size too small %d", sz)
```

Comment:
I understand what this check does but is it really necessary? Can we also remove this?

Reply:
It seems not a big deal. The check has already been there for a long time, and It doesn't do any harm. Let's keep it as it's.